vendor/bat-native-ads/src/bat/ads/internal/static_values.h

```diff
@@ -334,8 +334,30 @@ const std::map<int, std::vector<std::string>> kSupportedRegionsSchemas = {
   },
   {
     7, {
-      "101",  // World
```

Comment:
Removed 101 for World as it contains embargoed regions which would have appeared for NTP and Brave Ads